benchmarks/discrete_trajectory.cpp

```diff
@@ -34,6 +36,25 @@ not_null<std::unique_ptr<DiscreteTrajectory<World>>> CreateTrajectory(
   return trajectory;
 }
 
+// Creates a circular trajectory with the given number of steps.
+not_null<std::unique_ptr<DiscreteTrajectory<World>>> CreateCircularTrajectory(
+    int const steps) {
+  auto trajectory = make_not_null_unique<DiscreteTrajectory<World>>();
+  Instant t;
+  for (int i = 0; i < steps; i++, t += 1 * Second) {
+    const double sine = std::sin(i);
```

Comment:
We don't use the standard trigonometric functions, we use our wrappers in `quantities`.  Thus, with the proper `using` clauses:
```C++
constexpr AngularFrequency ω = 1 * Radian / Second;
for (Instant t = Instant{}; t < Instant{} + steps * 1 * Second; t += 1 * Second) {
  double const sin_ωt = Sin(ω * t);
  ...
```
Also, `const` to the East.